VK_App/VK_App/CustomNavigation/CustomInteractiveTransition.swift

```diff
@@ -0,0 +1,54 @@
+// CustomInteractiveTransition.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Жест в право (возвращает на предыдущий экран )
+final class CustomInteractiveTransition: UIPercentDrivenInteractiveTransition {
+    // MARK: - Public properties
+
+    var viewController: UIViewController? {
+        didSet {
+            let recognizer = UIScreenEdgePanGestureRecognizer(
+                target: self,
+                action: #selector(screenEdgeGestureRecognizerAction(_:))
+            )
+            recognizer.edges = [.left]
+            viewController?.view.addGestureRecognizer(recognizer)
+        }
+    }
+
+    var isStarted = false
```

Comment:
```suggestion
    var isStarted = false
```
марки а тут публичные